lib/less/parser/parser.js

```diff
@@ -325,6 +325,26 @@ var Parser = function Parser(context, imports, fileInfo) {
                     }
                 },
 
+                namedColor: function () {
+                    parserInput.save();
+                    var autoCommentAbsorb = parserInput.autoCommentAbsorb;
+                    parserInput.autoCommentAbsorb = false;
```

Comment:
Investigating #2802. Indeed that's mysterious. W/o `autoCommentAbsorb = false;` 
this `property: yes /* comment */; ` causes infinite parser loop (why?)
but otherwise  `property: red /* comment */; ` ovbiously is expected to fail. 
No ideas how to fix this so far.